### PR TITLE
New package: python3-celery-4.4.0

### PR DESCRIPTION
**Pull request by anjandev**
Needed for sr.ht.  #15784

### CODE REVIEW

**Comment by Chocimier** [comment]
Certainly it needs some depends. Set also `checkdepends="${depends}"`. Can be verified with `./xbps-src clean; ./xbps-src chroot` then import and do try anything with celery.

File: srcpkgs/python3-celery/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-celery'
+pkgname=python3-celery
+_pkgname=celery
```
Comment (Chocimier) [inline srcpkgs/python3-celery/template]: Used once, not needed.

**Comment by anjandev** [comment]
> Certainly it needs some depends. Set also `checkdepends="${depends}"`. Can be verified with `./xbps-src clean; ./xbps-src chroot` then import and do try anything with celery.

This is 100% correct. I overlooked this. Celery depends on Kombu which in turn requires an amqp transport (https://docs.celeryproject.org/projects/kombu/en/stable/introduction.html#features). Most of celery's dependencies are not completely packaged. Should the default amqp transport (py-amqp) be a depend of Kombu?

**Comment by Chocimier** [comment]
Kombu list amqp as its dependency, so yes.

**Comment by Chocimier** [comment]
Please put dependencies in same PR, one comit per template, so we'll see if it builds.

Set archs=noarch where appropriate.

File: srcpkgs/python3-celery/template (hunk 1 above)
Comment (anjandev) [reply srcpkgs/python3-celery/template]: removed

**Comment by anjandev** [comment]
@Chocimier Would you mind reviewing this again? I just initiated a rebuild and the package finally builds.

**Comment by Chocimier** [comment]
Running "First Steps with Celery" fails for me. How did you test celery?

Every module should depend on either another module or python3.

> Set also checkdepends="${depends}"

> Set archs=noarch where appropriate.

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**